pubsub/aws/snssqs/metadata.yaml

```diff
@@ -0,0 +1,160 @@
+capabilities:
+  - ttl
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: region
+    required: true
+    description: |
+      The AWS region where the SNS/SQS assets are located or be created in. See the regions product services page.
+      Ensure that SNS and SQS are available in that region.
+    url:
+      title: "Regions Product Services"
+      url: "https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/?p=ugi&l=na"
+    example: '"us-east-1"'
+    type: string
+  - name: endpoint
+    required: false
+    description: |
+      AWS endpoint for the component to use, to connect to emulators.
+      Do not use this when running against production AWS.
+    example: '"http://localhost:4566"'
+    type: string
+  - name: sessionToken
+    required: false
+    sensitive: true
+    description: |
+      AWS session token to use. A session token is only required if you are using 
+      temporary security credentials.
+    example: '"TOKEN"'
+    type: string
+  - name: messageVisibilityTimeout
+    required: false
+    description: |
+      Amount of time in seconds that a message is hidden from receive requests after 
+      it is sent to a subscriber.
+    type: number
+    default: '10'
+    example: '10'
+  - name: messageReceiveLimit
+    required: false
+    description: |
+      Maximun number of attempts the message will be re-delivered after processing failures.
+      The sqsDeadLettersQueueName is a SQS dead-letters queue to move the message to
+      once the maximun number of attempts have been reached.
+    type: number
+    default: '10'
+    example: '10'
+  - name: messageRetryLimit
+    required: false
+    description: |
+      Number of times to resend a message after processing of that message fails 
+      before removing that message from the queue.
+    type: number
+    default: '10'
+    example: '10'
+  - name: sqsDeadLettersQueueName
+    required: false
+    description: |
+      Name of the dead letters queue for this application.
+    example: '"myapp-dlq"'
+    type: string
+  - name: messageWaitTimeSeconds
+    required: false
+    description: |
+      The duration (in seconds) for which the call waits for a message to arrive
+      in the queue before returning. If a message is available, the call returns
+      sooner than messageWaitTimeSeconds. If no messages are available and the
+      wait time expires, the call returns successfully with an empty list of messages.
+    type: number
+    default: '1'
+    example: '1'
+  - name: messageMaxNumber
+    required: false
+    description: |
+      Maximum number of messages to receive from the queue at a time.
+    type: number
+    default: '10'
+    example: '10'
+  - name: fifo
+    description: |
+      Use SQS FIFO queue to provide message ordering and deduplication.
+      See SQS FIFO documentation further details.
+    url:
```

Comment:
Same question about url.

Reply:
@artursouza could you please clarify the ask here?

Reply:
@artursouza I've changed the url title. Hopefully it's more clear now.